lint/lint.py

```diff
@@ -413,8 +413,8 @@ def output_errors_text(errors):
     for error_type, description, path, line_number in errors:
         pos_string = path
         if line_number:
-            pos_string += " %s" % line_number
-        print("%s: %s %s" % (error_type, pos_string, description))
+            pos_string += ":%s" % line_number
+        print("%s: %s (%s)" % (pos_string, description, error_type))
```

Comment:
I guess it could be useful to be able to tell awk(1) to filter on error type, which is easier done with a format such as `file:col: ERROR TYPE: description`, so I could change it around so that your above example would be:

    2dcontext/drawing-text-to-the-canvas/2d.text.draw.fontface.html:31: SET TIMEOUT: setTimeout used; step_timeout should typically be used instead